dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessService.java

```diff
@@ -711,6 +700,14 @@ private ProcessInstance constructProcessInstance(Command command, String host) {
                 if (commandTypeIfComplement == CommandType.REPEAT_RUNNING) {
                     setGlobalParamIfCommanded(processDefinition, cmdParam);
                 }
+
+                // Recalculate global parameters after rerun.
+
+                processInstance.setGlobalParams(ParameterUtils.curingGlobalParams(
+                        processDefinition.getGlobalParamMap(),
+                        processDefinition.getGlobalParamList(),
+                        commandTypeIfComplement,
+                        processInstance.getScheduleTime()));
```

Comment:
What problem does this part of the code solve?

Reply:
It is useless, has been deleted, thanks for correcting